src/app/component/about-page/about-page.component.html

```diff
@@ -0,0 +1,162 @@
+<div class="cup-img"></div>
+
+<div class="container-2">
+
+  <div class="full-text-block">
+    <h2>
+      {{ 'about-us.intro.block-2.header' | translate }}
+    </h2>
+
+    <span>
+      {{ 'about-us.intro.block-2.text' | translate }}
+    </span>
+
+    <button>
+      {{ 'about-us.intro.block-2.button' | translate }}
+    </button>
+  </div>
+
+</div>
+
+<div class="steps-main-block">
+
+  <div class="dashed-line">
+    <div class="leaves"></div>
+  </div>
+
+  <h2>
+    {{ 'about-us.steps.title' | translate }}
+  </h2>
+
+  <div class="card-holder-odd first-card-holder">
+
+    <span class="number-title num1"></span>
+
+    <h3 class="card-h3">
+      {{ 'about-us.steps.block-1.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg1"></div>
+
+    <span class="main-card-text">
+      {{ 'about-us.steps.block-1.text' | translate }}
+    </span>
+
+    <a class="card-a" href="">
+      {{ 'about-us.steps.block-1.button' | translate }}
+    </a>
+
+  </div>
+
+  <div class="card-holder-even">
+
+    <span class="number-title num2"></span>
+
+    <h3 class="card-h3">
+      {{ 'about-us.steps.block-2.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg2"></div>
+
+    <span class="main-card-text">
+      {{ 'about-us.steps.block-2.text' | translate }}
+    </span>
+
+    <a class="card-a" id="link-2" href="">
+      {{ 'about-us.steps.block-2.button' | translate }}
+    </a>
+
+  </div>
+
+  <div class="card-holder-odd">
+
+    <span class="number-title num3"></span>
+
+    <h3 class="card-h3" id="heading-3">
+      {{ 'about-us.steps.block-3.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg3"></div>
+
+    <span class="main-card-text" id="text-3">
+      {{ 'about-us.steps.block-3.text' | translate }}
+    </span>
+
+    <a class="card-a" id="link-3" href="">
+      {{ 'about-us.steps.block-3.button' | translate }}
+    </a>
+
+  </div>
+
+  <div class="card-holder-even">
+
+    <span class="number-title num4"></span>
+
+    <h3 class="card-h3">
+      {{ 'about-us.steps.block-4.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg4"></div>
+
+    <span class="main-card-text" id="text-4">
+      {{ 'about-us.steps.block-4.text' | translate }}
+    </span>
+
+    <a class="card-a" id="link-4" href="">
+      {{ 'about-us.steps.block-4.button' | translate }}
+    </a>
+
+  </div>
+
+  <div class="card-holder-odd">
+
+    <span class="number-title num5"></span>
+
+    <h3 class="card-h3">
+      {{ 'about-us.steps.block-5.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg5"></div>
+
+    <span class="main-card-text" id="text-5">
+      {{ 'about-us.steps.block-5.text' | translate }}
+    </span>
+
+    <a class="card-a" href="" id="link-5">
+      {{ 'about-us.steps.block-5.button' | translate }}
+    </a>
+
+  </div>
+
+
+</div>
+
+<footer>
+  <app-footer></app-footer>
+  <div class="copyright">
+    &copy; Copyright {{ actualYear }}. Green City.
+  </div>
```

Comment:
generally delete all unneeded enters as line 2;
change class, id and picture name without numbers and tag names

Reply:
done